src/Sentry.Unity.Native/SentryNative.cs

```diff
@@ -0,0 +1,45 @@
+using Sentry.Extensibility;
+
+namespace Sentry.Unity.Native
+{
+    /// <summary>
+    /// Access to the Sentry native support of the sentry-native SDK.
+    /// </summary>
+    public static class SentryNative
+    {
+        /// <summary>
+        /// Configures the native SDK.
+        /// </summary>
+        /// <param name="options">The Sentry Unity options to use.</param>
+        public static void Configure(SentryUnityOptions options)
+        {
+            if (options.WindowsNativeSupportEnabled)
+            {
+                SentryNativeBridge.Init(options.Dsn!, options.Release!);
```

Comment:
if `!` is set here I assume the method doesn't accept nullable. So either we need to check for null before calling, or make the `Init` method take null and deal with it later.
If we can't have `Init` do its thing with null arguments, we should just check for those values here before we call and log in case we're not calling due to missing options.

Reply:
Note: DSN is not null because `options.Validate()` must have returned true in SentryInitialization.cs for this to be called at all. I've added a comment.